.github/workflows/ci.yml

```
@@ -1,86 +1,347 @@
-name: Code Scanning, Linting, and Testing
+name: Omega_KG CI Pipeline
 
 on:
+  push:
+    branches: [ "beta", "main" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)